lms/djangoapps/class_dashboard/tests/test_views.py

```diff
@@ -88,7 +88,8 @@ def test_section_problem_grade_distribution_no_access(self, has_access):
 
         self.assertEqual("{\"error\": \"Access Denied: User does not have access to this course\'s data\"}", response.content)
 
-    def test_sending_deprecated_id(self):
+    @patch('class_dashboard.dashboard_data.settings')
+    def test_sending_deprecated_id(self, mock_settings):
```

Comment:
I had to mock settings here, because the three data requesting methods referenced in this test call the data_dashboard methods I switched to using the client which fetch the API url and token from settings in order to initialize the client.  Is there another way I should be handling settings in tests?